applications/registration-react/src/auth/auth-service.js

```diff
@@ -67,6 +55,8 @@ export function login({ readOnly = false }) {
   kc.login({ redirectUri: window.location.origin, idpHint });
 }
 
+export const isAuthorized = () => kc && kc.authenticated;
```

Comment:
Move this check into hasOrganizationAdminPermission function

Reply:
fikset